core/services/ocr2/plugins/ocr2vrf/internal/ocr2vrf_integration_test.go

```diff
@@ -312,7 +312,6 @@ func TestIntegration_OCR2VRF(t *testing.T) {
 	t.Log("Adding bootstrap node job")
 	err = bootstrapNode.app.Start(testutils.Context(t))
 	require.NoError(t, err)
-	defer bootstrapNode.app.Stop()
```

Comment:
No `t.Cleanup`?

Reply:
These are actually redundant because `(*TestApplication).Start` already registers them via `t.Cleanup` internally.

Reply:
(linter was just complaining about the unchecked error - which Start also handles)

Reply:
Ah that's great 😄